vignettes/z_code_for_article_in_musik_und_internet.Rmd

```diff
@@ -0,0 +1,119 @@
+---
+title: "Code for Article 'psychTestR - Ein R-Paket für (musik-) psychologische Tests und Fragebogeninventare'"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Code_for_article_in_musik_und_internet}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+The following code is taken from the article "*psychTestR* - Ein R-Paket für (musik-) psychologische Tests und Fragebogeninventare" in "Musik und Internet. Musikalische Phänomene populärer Kulturen" (2022) by Nicolas Ruth & Peter Moormann (see [article](https://.html)). Feel free to copy and modify it according to your needs. <!-- This link needs to be specified. Viola informs Peter as soon as she knows the link. -->
+
+```r
+# Beispielcode  ------------------------------------------------------
+ 
+install.packages("devtools")
```

Comment:
I don't think you should put 'install packages' commands in app.R, because they will run every time you run the app. Maybe comment them out?

Reply:
I see your point. I've added a separate Rmd section.